jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -34,6 +35,7 @@ class Catalogs(PluginTemplateMixin, ViewerSelectMixin, HasFileImportSelect, Tabl
     catalog_selected = Unicode("").tag(sync=True)
     results_available = Bool(False).tag(sync=True)
     number_of_results = Int(0).tag(sync=True)
+    max_sources = IntHandleEmpty(1000).tag(sync=True)
```

Comment:
this currently isn't exposed in the user API - which might be the way to go if it isn't something that we can generalize to other catalogs as well.  If it is gaia-specific, then we may want a `gaia_` prefix to make that clear.

Reply:
Never mind, as mentioned above it does not appear this option exists for SDSS so I just renamed the variable to `max_gaia_sources`.